docker-for-azure/faqs.md

```diff
@@ -73,7 +73,7 @@ $ sudo ping 10.0.0.4
 
 
 ## What are the different Azure Regions?
-All regions can be found here: [Microsoft Azure Regions](https://azure.microsoft.com/en-us/regions/)
+All regions can be found here: [Microsoft Azure Regions](https://azure.microsoft.com/en-us/regions/).
```

Comment:
Oh wow, you are right. Sometimes it's hard to read details in fonts! Sorry about that.